serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala

```diff
@@ -196,5 +197,23 @@ class SerializabilityCheckerCompilerPluginComponent(
         else
           tp.parents.flatMap(findSuperclassAnnotatedWithSerializabilityTrait).headOption
       }
+
+      /*
+      It might happen that scala compiler translates the "[_]" wildcard from typeArgs
+      not to the "scala.Any" type, but to a temporary compiler-specific type.
+      String representation for such type ends with "._$<DIGIT>" (e.g. "._$2")
+      To catch such situations and not raise false error - we have to check it.
+      We should ignore scala.Any and other possible ignoredTypes in such cases.
+      In particular, mentioned problem might occur when using ActorSystem[_] implicitly as parameter.
+      For example, such scenario occurs when using object of type ActorSystem[_] in method calls
+      that in fact require akka.actor.ClassicActorSystemProvider (which is a supertype for ActorSystem).
+       */
+      private def typeArgsAreHiddenIgnoreTypes(typ: global.Type): Boolean =
+        if (typ.typeSymbol.fullName.matches(".+\\._\\$\\d"))
```

Comment:
Check `raw"..."` interpolator

Reply:
I checked it, but I don't know how can it help with regular expression? Is there something wrong with current regex string `".+\\._\\$\\d"` ?

Reply:
Oh! CMIIW... can backslashes remain un-escaped within `raw"..."` string? or maybe I confused it with `r"..."` from Python ;p